src/Microsoft.VisualStudio.Services.Agent/Blob/CustomerIntelligenceTelemetrySender.cs

```diff
@@ -81,5 +100,21 @@ public async Task CommitTelemetry(Guid planId, Guid jobId)
             };
             await _ciClient.PublishEventsAsync(new [] { ciEvent });
         }
+
+        public Dictionary<string, object> GetTelemetryDownload(Guid planId, Guid jobId)
```

Comment:
This isn't so much a download, as adding telemetry right?  Perhaps "AddArtifactData"?

Reply:
This returns the telemetry we've collected while downloading files. Not sure how `AddArtifactData` would describe this better.

Reply:
How about "GetArtifactDownloadTelemetry"?